src/Loader.php

```diff
@@ -221,16 +221,17 @@ public function __construct(
 
         /** The specified vendor directory doesn't exist or isn't readable. */
         if (!is_dir($VendorPath) || !is_readable($VendorPath)) {
-            if (isset($_SERVER['DOCUMENT_ROOT'], $_SERVER['SCRIPT_NAME'])) {
-                /** Safeguard for symlinked installations. */
-                $VendorPath = $this->buildPath(dirname($_SERVER['DOCUMENT_ROOT'] . $_SERVER['SCRIPT_NAME']) . DIRECTORY_SEPARATOR . 'vendor', false);
-
-                /** Eep.. Still not working. Generate exception. */
-                if ($VendorPath === '' || !is_dir($VendorPath) || !is_readable($VendorPath)) {
-                    throw new \Exception('Vendor directory is undefined or unreadable.');
-                }
-            } else {
+            if (! isset($_SERVER['DOCUMENT_ROOT'], $_SERVER['SCRIPT_NAME'])) {
```

Comment:
The complaint from PHP-CS-Fixer:
```
   1) src/Loader.php (no_trailing_whitespace, statement_indentation, return_type_declaration)
```
..can be fixed by dropping the space between the `!` and the `isset`. A minor detail, but that's code-style. '^.^